igvf_utils/__init__.py

```diff
@@ -69,8 +69,8 @@
 #: A hash of known hosts one can connect to, where the key can be passed to the `dcc_mode` argument
 #: when instantiating the `connection.Connection` class.
 DCC_MODES = {
-    DCC_DEV_MODE: {"url": "https://test.encodedcc.org"},
-    DCC_PROD_MODE: {"url": "https://www.encodeproject.org"}
+    DCC_DEV_MODE: {"url": ""},
+    DCC_PROD_MODE: {"url": "https://igvfd-dev.demo.igvf.org/"}
 }
 
 ENCODE_PROD_S3BUCKET="encode-files"
```

Comment:
either give up on buckets or rename to IGVF bucket?

Reply:
This is also where the modes are defined, should I just remove those? I turned "dev" to a blank string so it won't be functional, and "prod" changed to the alpha demo.

I could rename the variables and replace the values with blank strings. I was hesitant to completely delete all the implementation of file uploading since we would likely have to add it back later.